common/stream-model.cpp

```diff
@@ -1041,6 +1041,22 @@ namespace rs2
         ImGui::EndChild();
     }
 
+    std::string stream_model::format_value(rs2_frame_metadata_value& md_val, rs2_metadata_type& attribute_val) const
+    {
+        if (should_show_in_hex(md_val))
+            return rsutils::string::from() << "0x" << std::hex << attribute_val; // return value as hex
+        return rsutils::string::from() << attribute_val;
+    }
+
+    bool stream_model::should_show_in_hex(rs2_frame_metadata_value& md_val) const
+    {
+        static std::unordered_set< rs2_frame_metadata_value > show_in_hex = {};
```

Comment:
Please add a comment
`place in the SET metadata type you wish to display in HEX format`